src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -0,0 +1,571 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { AccountTypes, ExchangeOffer, NftList } from 'etherspot';
+import { TokenListToken } from 'etherspot/dist/sdk/assets/classes/token-list-token';
+import { ethers } from 'ethers';
+import debounce from 'debounce-promise';
+
+// Types
+import { IPlrDaoStakingMembershipBlock } from '../../types/transactionBlock';
+
+// Components
```

Comment:
please keep it consistent on your own code mate, thanks!
```suggestion
// components
```